pynest/nest/lib/hl_api_info.py

```diff
@@ -271,14 +271,15 @@ def SetStatus(nodes, params, val=None):
     if len(nodes) == 0:
         return
 
-    n0 = nodes[0]
     params_is_dict = isinstance(params, dict)
     set_status_nodes = isinstance(nodes, nest.NodeCollection)
-    set_status_local_nodes = set_status_nodes and n0.get('local')
+    set_status_local_nodes = set_status_nodes and nodes.get('local')
```

Comment:
Now that we use the whole NodeCollection here, we should check that all are local.
```suggestion
    set_status_local_nodes = set_status_nodes and all(nodes.local)
```
Or should we still check only the first node?